advisories/2.4.1/BRSA-6zgfowt0osgt.toml

```diff
@@ -0,0 +1,18 @@
+[advisory]
+id = "BRSA-6zgfowt0osgt"
+title = "libexpat CVE-2024-45490"
+cve = "CVE-2024-45490"
+severity = "high"
+description = "An issue was discovered in libexpat before 2.6.3. xmlparse.c does not reject a negative length for XML_ParseBuffer."
+
+[[advisory.products]]
+package-name = "libexpat"
+patched-version = "2.6.3"
+patched-release = "0"
+patched-epoch = "0"
+
+[updateinfo]
+author = "yeazelm"
+issue-date = 2024-09-10T02:19:24Z
```

Comment:
This core kit was released at `2024-09-09T20:04:49Z`, let's use that issue-date.